Dizzy/Screens/SignUp/SignUpWithDizzyVC.swift

```diff
@@ -50,7 +50,7 @@ final class SignUpWithDizzyVC: UIViewController, KeyboardDismissing, LoadingCont
     }
     
     private func setupNavigationView() {
-        self.navigationItem.title = "Sign Up".localized
+        self.navigationItem.title = "".localized
```

Comment:
same here, if we dont want title we should not set it

Reply:
done